docling_serve/response_preparation.py

```diff
@@ -69,9 +72,13 @@ def _export_document_as_content(
         if export_doctags:
             document.doctags_content = new_doc.export_to_document_tokens()
     elif conv_res.status == ConversionStatus.SKIPPED:
-        raise HTTPException(status_code=400, detail=conv_res.errors)
+        raise HTTPException(
```

Comment:
On the single document, this makes perfect sense.
But this might not be the expected behavior when the user provides multiple document.

In this proposal, if 10 documents are provided and 1 fails, it will raise the error and produce no results.

In the latter, it would be better to have some manifest response file in the zip, where the system lists the result for each file.